external/llvm-external-projects/torch-mlir-dialects/test/lit.site.cfg.py.in

```diff
@@ -9,6 +9,7 @@
 import sys
 
 config.torch_mlir_dialects_obj_root = "@TORCH_MLIR_DIALECTS_BINARY_DIR@"
+config.torch_mlir_obj_root = "@TORCH_MLIR_BINARY_DIR@"
```

Comment:
We should probably use a different variable here, to point to the directory where the `torch-mlir-dialects-opt` is found (`llvm-project` build dir for in-tree, `torch-mlir` build dir for out-of-tree).